pom.xml

```diff
@@ -4,26 +4,64 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.example</groupId>
-    <artifactId>test-automation</artifactId>
-    <packaging>pom</packaging>
+    <groupId>pl.bugdemons</groupId>
+    <artifactId>test-automation-template</artifactId>
+    <packaging>jar</packaging>
     <version>1.0-SNAPSHOT</version>
 
     <properties>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
+
         <io.swagger.version>1.6.2</io.swagger.version>
         <google.gson.version>2.8.5</google.gson.version>
         <javax.annotation.version>1.3.2</javax.annotation.version>
-        <openapi.generator.version>5.0.0</openapi.generator.version>
+
+        <spring.version>5.3.9</spring.version>
+        <openapi.generator.version>5.1.0</openapi.generator.version>
+        <junit.version>5.8.1</junit.version>
+        <log4j2.version>2.14.1</log4j2.version>
+        <jackson.version>2.12.5</jackson.version>
     </properties>
 
     <dependencies>
         <!-- lombok -->
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.18.12</version>
+            <version>1.18.20</version>
+        </dependency>
+
+        <!-- logging -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-slf4j-impl</artifactId>
+            <version>${log4j2.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.32</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
```

Comment:
te zależności do log4j w dół nie są zbędne?

Reply:
etapami to dodawałem, zerknę czy to wymagane

Reply:
tylko zależy o co pytasz (github troche kijowo wyświetla te komentarze) bo te 3 poniżej są określane jako niezbędne do parsowania yaml dlatego je dodałem